### DISCUSSION TITLE
Error with tidyverse and googledrive package

### DISCUSSION THREAD

**Discussion by kikislater** [eblondel/geoflow#331] [General]
Sometimes connection through R to google drive doesn't work. Some errors are reported like : 

```
Loading required package: googledrive
Error in eval(parse(text = paste0("self$handler(", handler_params, ")"))) :  
attempt to apply non-function
```
or
 
```
Loading required package: googledrive
Error in self$handler(email = "xxxx@gmail.com", token = "") : 
  Can't get Google credentials.
ℹ Are you running googledrive in a non-interactive session? Consider:
• Call `drive_deauth()` to prevent the attempt to get credentials.
• Call `drive_auth()` directly with all necessary specifics.
ℹ See gargle's "Non-interactive auth" vignette for more details:
ℹ <https://gargle.r-lib.org/articles/non-interactive-auth.html>
```

or 

![image](https://github.com/r-geoflow/geoflow/assets/13238064/14f0a291-71d8-4b9c-a3e2-59d51d4dde45)

or 

![image](https://github.com/r-geoflow/geoflow/assets/13238064/3ad3ebd0-209b-4d93-a8bc-ae9b03188440)

It seems difficult for the user to connect anymore to google drive. I think it should be documented ... User have to go to: 
```
Google API Console => identifiants => API keys
```
And create a new API key

Official documentation : https://developers.google.com/identity/protocols/oauth2

**Comment by eblondel** [comment]
did you try to update tidyverse package? what are the versions of tidyverse / google drive on your machine?

**Comment by kikislater** [comment]
Yes but it's not me at this time... Last time it was a mess to find it ! 
As it's a discussion, it's not a bug but a feedback ...

**Reply by eblondel** [reply]
@BastienIRD and @juldebar have experimented the same issue, to keep track of it, and see what we can do.
